templates/zerver/landing_nav.html

```diff
@@ -7,174 +7,252 @@
+                          fill="white" />
+                    </g>
+                    <defs>
+                        <clipPath id="clip0_555_14092">
+                            <rect width="108" height="28" fill="white" />
+                        </clipPath>
+                    </defs>
+                </svg>
+            </a>
+        </div>
+        <div class='top-menu__items-group-1'>
+            <div class='top-menu__item top-menu__tab'>
+                <div class="top-menu__tab-unselect"></div>
+                <input type="radio" name="top-menu-tabs" class="top-menu__tab-input" id="top-menu__tab-product" />
+                <label for="top-menu__tab-product">Product</label>
+                <div class="top-menu__submenu">
+                    <div class="top-menu__submenu-column">
                         <a href="https://zulip.com">Home</a>
-                    </li>
-                    <li>
-                        <a href="https://zulip.com/why-zulip">Why Zulip</a>
-                    </li>
-                    <li>
-                        <a href="https://zulip.com/features/">Features</a>
-                    </li>
-                    <li>
-                        <a href="https://zulip.com/self-hosting/">Self-hosting</a>
-                    </li>
-                    <li>
-                        <a href="https://zulip.com/security/">Security</a>
-                    </li>
-                    <li>
+                        <a href="/why-zulip">Why Zulip</a>
+                        <a href="/features/">Features</a>
+                        <a href="/security/">Security</a>
                         <a href="/integrations/">Integrations</a>
-                    </li>
-                    <li>
-                        <a href="/api/">REST API</a>
-                    </li>
-                </ul>
+                        <a href="/apps">Apps</a>
+                        <a href="https://chat.zulip.org/">Demo organization</a>
+                    </div>
```

Comment:
I think the same comment I had for the footer applies here -- these are menus that should have a `ul/li` structure, rather than being a bunch of `a` tags inside a `div` with styling to put each `a` tag on its own line.

(And we also want the absolute links, so we're mostly going to revert this to roughly the old HTML structure, albeit with new class names and of course differently organized content)

Reply:
Done.